### DISCUSSION TITLE
Step 5 - I cant apply to suggested changes

### DISCUSSION THREAD

**Discussion by Raphaellaucene** [skills/.github#408] [Review pull requests]
Hi! After to added a single comments, I dont know what I have to do now.. this step tell me to:

Click Commit suggestion.
Type a commit message.
Click Commit changes.

but, this options doesnt appear.. I dont know how I have to do.. Can you help me? thanks.

**Comment by KevinRaffay** [comment]
Same

**Comment by lgaceta** [comment]
Me too.

**Comment by Shashanka10** [comment]
Guys I know where you would make mistake
1) go to pull request 
2) files changed
3)go to index.html
4) click the blue plus button  on the html tag that you want to suggest changes on ( blue plus button appears) when u over on html tag
![sug1](https://github.com/skills/.github/assets/71580178/d6b87f6c-255b-40a0-a9f4-93523d82ad44)
5) then u will see a message box where u have a button to add suggestion , all you need to do is click on it
![sug2](https://github.com/skills/.github/assets/71580178/2842aeaf-d833-491d-b670-ccfa36a9080f)
6) after that you will see a suggestion message that appears automatically 
7) I have edited the original header tag information with the added information as reflected by yellow color.
![sug3](https://github.com/skills/.github/assets/71580178/52103804-36df-4dae-9234-48f722c8c5b6)
7) I have already removed emoij that were there at first, so this is the second suggestion I brought here to show u guys how it works. Suggest what you want to change in  a html tag as shown in the pic above (yellow color text is the suggested message) and give a suggestion message at the last line , below `` to make it work.
8) after this u will get a commit a suggestion box on next step

**Reply by hyphena** [reply]
It worked, thank you ^^

**Reply by marianof2000** [reply]
Thanks a lot!! :)

**Reply by hevt** [reply]
you're a star! Thanks!

**Comment by Raphaellaucene** [comment]
Its works kk.. thanks man, now, I can move to step 6.. thanks a lot bro..
